FirebaseSessions/Tests/Unit/IdentifiersTests.swift

```diff
@@ -60,15 +68,16 @@ class IdentifiersTests: XCTestCase {
 
     let firstSessionID = identifiers.sessionID
     XCTAssert(isValidSessionID(identifiers.sessionID))
-    XCTAssert(identifiers.previousSessionID.count == 0)
+    XCTAssertNil(identifiers.previousSessionID)
 
     identifiers.generateNewSessionID()
 
     XCTAssert(isValidSessionID(identifiers.sessionID))
-    XCTAssert(isValidSessionID(identifiers.previousSessionID))
+    XCTAssert(isValidSessionID(identifiers.previousSessionID ?? ""))
 
     // Ensure the new lastSessionID is equal to the sessionID from earlier
-    XCTAssert(identifiers.previousSessionID.compare(firstSessionID) == ComparisonResult.orderedSame)
+    XCTAssert((identifiers.previousSessionID ?? "").compare(firstSessionID) == ComparisonResult
+      .orderedSame)
```

Comment:
General comment across - Can we use `XCTAssertEqual` for object comparison? It is a bit more comprehensive in comparison that direct comparison of strings.

Reply:
Huh it seems they fixed String equality in Swift - I always had problems using AssertEqual for strings in ObjC